### PR TITLE
Bug fix in nFields

### PR DESCRIPTION
**Pull request by bkochuna**
Description:
Quoted strings were truncating the number of fields by 1.

Test updated to test quoted strings. This had been commented out. Quoted
strings also tested for multiples.

### CODE REVIEW

**Comment by bkochuna** [comment]
ping @stimpsonsg or @aarograh this has just been sitting for awhile. should be a pretty quick review. If either of you do not have time though I can assign to someone else.

File: src/IO_Strings.f90 [hunk 1]
```diff
@@ -684,6 +683,14 @@ PURE FUNCTION nFields_char(aline)  RESULT(nfields)
             READ(aline(i+1:multidcol-1),*,IOSTAT=ioerr) nmult
             IF(ioerr /= 0) nmult=1
             n=n+(nmult-1)*2
+
+            !If we are multiplying a quoted string need to subtract 1.
```
Comment (stimpsonsg) [inline src/IO_Strings.f90]: When you saying "multiplying a quoted string"...do you mean like 2*"some string"?

Comment (bkochuna) [reply src/IO_Strings.f90]: yes. exactly.

Comment (bkochuna) [reply src/IO_Strings.f90]: IF you look farther down near line 229 this test already existed (didn't need to add it). The problem was that the previously when that was made to work it broke the cases that were commented out.

I found an alternative way of implementing the routine such that both sets of ASSERTS could pass.

Make sense?

Comment (stimpsonsg) [reply src/IO_Strings.f90]: I see...makes sense.

**Pull request merged**